mllib/src/main/scala/org/apache/spark/ml/ann/BreezeUtil.scala

```diff
@@ -55,7 +55,7 @@ private[ann] object BreezeUtil {
    * @param y y
    */
   def dgemv(alpha: Double, a: BDM[Double], x: BDV[Double], beta: Double, y: BDV[Double]): Unit = {
-    require(a.cols == x.length, "A & b Dimension mismatch!")
+    require(a.cols == x.length, "A & x Dimension mismatch!")
```

Comment:
Y'know, while we're at it, the matrix args in this whole class don't match the docs. The arg here and similarly elsewhere can be called "A" not "a".

Also this should check that `A.rows == y.length` for consistency.